pkgs/development/idris-modules/build-idris-package.nix

```diff
@@ -4,12 +4,12 @@
 #       name and src.
 { stdenv, idris, gmp }: args: stdenv.mkDerivation ({
   preHook = ''
-    mkdir idris-libs
+    mkdir -p idris-libs
```

Comment:
What did this fix? It should be very rare for your current directory to not exist.